restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -203,7 +203,9 @@ public static ValueModifier colToValue(String name, Object value, AbstractTable
   /** Converts an incoming JSON value into a Java type suitable for the given column type. */
   @SuppressWarnings("unchecked")
   public static Object toCqlValue(Column.ColumnType type, Object jsonValue) {
-
+    if (jsonValue == null) {
```

Comment:
Looks like this is sufficient as the actual fix: driver(s) seem capable of then translating value as necessary for backend access (or failing if "null"s are actually not allowed)

Reply:
Yep this seems enough..